src/pages/api/card/getIds.js

```diff
@@ -0,0 +1,32 @@
+import { withSessionRoute } from "src/lib/utils/session";
+import { getCardsByIds } from "../../../../server/mongodb/actions/Card";
+
+// @route   GET api/card/getIds
+// @desc    Gets cards by their ids
+// @access  Public
+const handler = async (req, res) => {
+  try {
+    // const userId = req.session.user.id;
+    let cards;
+    if (req.body) {
+      let ids = req.body;
```

Comment:
(just a heads up that if you change this to be a `GET` route, you might have to change this code to use `req.query`)

Reply:
Meant to rename this to POST, the number of ids that should be accepted by the endpoint exceeds the data limit of a GET request. Do you think I should rename the file?